src/main/java/frc/robot/commands/states/climbing/ClimberDeploying.java

```diff
@@ -47,6 +50,7 @@ public void initialize() {
       globalAlgaeIntake.setAlgaePivotAngle(Constants.constAlgaeIntake.CLIMB_DEPLOY_POSITION);
       globalElevator.setPosition(Constants.constElevator.ZEROED_POS);
       globalClimber.setClimberMotorVelocity(Constants.constClimber.CLIMBER_MOTOR_DEPLOYING_VELOCITY);
+      globalAlgaeIntake.setAlgaeIntakeMotor(Constants.constAlgaeIntake.CLIMBER_ALGAE_VELOCITY);
```

Comment:
lets move this above the pivot so it starts spitting before moving.

we should also set it back to zero in execute to save power